lib/site_prism/waiter.rb

```diff
@@ -13,5 +13,12 @@ def self.wait_until_true(wait_time = Capybara.default_max_wait_time)
 
       raise SitePrism::TimeoutException, wait_time
     end
+
+    def self.wait_with_capybara(timeout: nil, or_raise: nil)
+      timeout = Capybara.default_max_wait_time if timeout.nil?
+      result = Capybara.using_wait_time(timeout) { yield }
```

Comment:
This won't work correctly if the user is using Capybaras "threadsafe" mode - https://github.com/teamcapybara/capybara#threadsafe - not sure if that's a concern or not.  Could be fixed by calling on the session or by leaving methods passing the wait time to each find operation